apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/hooks/useTransactionMonitoringAlertsLogic/useTransactionMonitoringAlertsLogic.tsx

```diff
@@ -29,6 +30,8 @@ export const useTransactionMonitoringAlertsLogic = () => {
         ?.sort((a, b) => (a?.id === session?.user?.id ? -1 : b?.id === session?.user?.id ? 1 : 0)),
     [assignees, session?.user?.id],
   );
+  const { data: labels } = useAlertLabelsQuery();
+  const sortedLabels = useMemo(() => labels?.sort(), [labels]);
```

Comment:
`Array.prototype.sort` mutates.
```suggestion
  const sortedLabels = useMemo(() => labels?.slice()?.sort(), [labels]);
```

Reply:
Also is there a need for the labels to be sorted?

Reply:
There's no clear reason for sorting the labels. Just kept the logic we have with `assignees`, and it's kinda makes sense to sort the label, but not important.

Reply:
Note the assignees is sorted so the signed in user is always at the top.